namd/src/colvarproxy_namd.C

```diff
@@ -604,13 +611,48 @@ void colvarproxy_namd::calculate()
   reduction->submit();
   #endif
 
+  if (atom_list_frequency() > 0) {
+    if (((cvm::step_relative()+1) % atom_list_frequency()) == 0) {
+      // Before all-atom evaluation
+      update_requested_atoms();
+    }
+    if ((cvm::step_relative() % atom_list_frequency()) == 0) {
+      // After all-atom computation
```

Comment:
I may need to implement this in https://github.com/Colvars/colvars/pull/783 as well. Why is `update_requested_atoms()` called twice?

Reply:
Those two requests happen at consecutive steps, first to recall all atoms that have positive refcount, then at the next step to prune those that don't.